examples/README.md

```diff
@@ -0,0 +1,4 @@
+## aws-lambda-powertools-examples
+
+This directory holds example projects demoing different components of the java powertools.
```

Comment:
replace java powertools with "Lambda Powertools for Java" or "Lambda Powertools"